docs/en_US/Release.rst

```diff
@@ -5,6 +5,99 @@
 ChangeLog
 =========
 
+Release 2.0 - 11/1/2021
+=======================
+
+Major updates
+-------------
+
+Neural architecture search
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Support an improved NAS framework: Retiarii (experimental)
+
+  * Feature roadmap (TODO)
+  * Related issues and pull requests (label: retiarii-2.0)
```

Comment:
change the (label: retiarii-2.0) to an url for easy access, something like below:

[Related issues and PR](https://github.com/microsoft/nni/issues?q=label%3Aretiarii-2.0+)

Reply:
Same comments to all the issues, add like to them. i.e. #3221

Reply:
updated and added.